README.md

```diff
@@ -1 +1,10 @@
-# mint_login-clone
\ No newline at end of file
+# Course: HTML & CSS3
+## Second microverse project
+
+This is the first version of the second project for the Microverse training program.
+It was developed by Nico and Oscar (the dream team) using basic html tags, form tags,
+image map tags and CSS styling.
+
+It's a functional copy of the site [Mint.com](https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D08497889761544046421391270368961118431%257CMCAID%253D2E930D15050307F2-400011802000029C%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1562794843%26ivid%3D538f0975-5631-4a09-93f2-0c8809828c71). Not every function was
+repicated, but we tried to replicate the structure, the look&feel and validation of the
+original site.
```

Comment:
`AWESOME`

Nice job writing a descriptive `README.md` file for describing what the project does. You can even add other information like live link and a screenshot of the webpage. :+1: :star2: